tests/platform_tests/test_advanced_reboot.py

```diff
@@ -62,15 +85,29 @@ def test_fast_reboot_from_other_vendor(duthosts,  rand_one_dut_hostname, request
     flush_dbs(duthost)
     advancedReboot.runRebootTestcase()
 
+
 @pytest.mark.device_type('vs')
-def test_warm_reboot(request, get_advanced_reboot, verify_dut_health,
-    advanceboot_loganalyzer, capture_interface_counters):
+def test_warm_reboot(request, testing_config, get_advanced_reboot, verify_dut_health, duthosts, advanceboot_loganalyzer, 
+    capture_interface_counters, toggle_all_simulator_ports, rand_selected_dut, toggle_simulator_port_to_upper_tor):
     '''
     Warm reboot test case is run using advacned reboot test fixture
 
     @param request: Spytest commandline argument
     @param get_advanced_reboot: advanced reboot test fixture
     '''
+    testing_mode = testing_config
+    if testing_mode == DUAL_TOR_MODE:
+        duthost = rand_selected_dut
+        toggle_all_simulator_ports(LOWER_TOR)
+        check_result = wait_until(120, 10, 10, check_mux_status, duthosts, LOWER_TOR)
+        validate_check_result(check_result, duthosts, get_mux_status)
+        mux_list = show_muxcable_status(duthost)
+        toggle_mux_size = len(mux_list) / 2
+        for i in range(toggle_mux_size):
+            itfs, _ = rand_selected_interface(duthost)
+            # Set upper_tor as active
```

Comment:
minor: comment is misleading. There isn't a active/standby tor in this case, right? We have a mixed distribution.

Reply:
Correct, updated comment